.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 *.bak
+.venv
```

Comment:
Also please make sure you rebase this PR with the previous one so that it only contains changes related to formatting.